README.md

```diff
@@ -1,6 +1,8 @@
 <img src="images/razee_icon.png">
 
-[Razee](http://www.razee.io/) is an open-source project that was developed by IBM to automate and manage the deployment of Kubernetes resources across clusters, environments, and cloud providers, and to visualize deployment information for your resources so that you can monitor the rollout process and find deployment issues more quickly.
+[Razee](http://www.razee.io/) is an open-source project that was developed by IBM to automate and manage the deployment of Kubernetes resources across clusters, environments, and cloud providers, and to visualize deployment information for your resources so that you can monitor the rollout process and find deployment issues more quickly. Check out some of its tools in this video overview: 
```

Comment:
Lint check failed. Need to remove the trailing space on line 3

`README.md: 3: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]`